lms/djangoapps/verify_student/management/commands/tests/test_backfill_sso_verifications_for_old_account_links.py

```diff
@@ -0,0 +1,60 @@
+"""
+Tests for management command backfill_sso_verifications_for_old_account_links
+"""
+
+from mock import patch
+
+from django.core.management import call_command
+from django.core.management.base import CommandError
+from django.dispatch import receiver
+
+from lms.djangoapps.program_enrollments.management.commands.tests.utils import UserSocialAuthFactory
+from lms.djangoapps.verify_student.models import SSOVerification
+from third_party_auth.tests.testutil import TestCase
+
+
+class TestBackfillSSOVerificationsCommand(TestCase):
```

Comment:
I think you can add another test case for learners who already have SSOVerification records. Make sure no additional SSO records created